demo/scripts/load-driver-image-into-kind.sh

```diff
@@ -27,6 +27,16 @@ set -o pipefail
 
 source "${CURRENT_DIR}/common.sh"
 
-kind load docker-image \
-	--name "${KIND_CLUSTER_NAME}" \
-	"${DRIVER_IMAGE}"
+# Work around kind not loading image with podman
+if [[ "${DOCKER}" == "podman" ]]; then
+	IMAGE_ARCHIVE=driver_image.tar
```

Comment:
should we use a file in `/tmp`.

Also in theory this workflow can be used for both runtimes. Should we just use the one instead of having a conditional block?

Reply:
Normally the file will be removed right after loading into Kind, so I don't think it matters.

Correct, the flow can be used for both runtimes, but I couldn't test extensively on Docker so preferred to make as little changes as possible at the moment.